docs/advanced/command-line/files/wrangle-properties.md

```diff
@@ -0,0 +1,152 @@
+
+- filters and criteria. These two properties describe the [User Context](../../../../guides/setting-priorities/intro/), i.e. your preferences over the characteristics of the end product. The filters are hard constraints on the end product, meaning that the end product should satisfy these constraints. 
+
+    Criteria and filters can be of any of the following types:
+
+    Standalone, that require no parameters for their evaluation.
+
+        - completeness. The fraction of the values that are null or empty.
+        - tuple_completeness. The fraction of the tuples with non-null, non-empty values. Can only be evaluated on a table.
+        - numeric. The fraction of the values that are numbers.
+        - datetime. The fraction of the values that represent a date or a time.
+        - is_email. The fraction of the values that are emails.
+        - is_uri. The fraction of the values that are uris.
+        - contain_numbers. The fraction of values that contain any number.
+        - contain_letters. The fraction of values that contain any letter.
+        - contain_punctuation. The fraction of values that contain punctuation characters, i.e. characters that are not letters, or numbers, or whitespace.
+        - contain_email. The fraction of the values that contain emails.
+        - contain_uri. The fraction of the values that contain uris.
+
+    Data context-related, that require the presence of data context.
+
+        - data_context_completeness. The fraction of the values that are included in the Master Data or the Example values.
+        - data_context_consistency. The fraction of the values that are included in the Reference Data or the Example values.
+
+    Feedback-related, that require the presence of user feedback.
+
+        - relevance. The fraction of the values that the user has marked as relevant.
+
+    > An example of a filter is completeness:product.name:no, which means that the attribute name of the end product table product must be complete. The no in the end is optional, and means that the filter is not inverted. Criteria are defined in the same way, with the addition of weights. An example definition of criteria is: completeness:product:0.5:yes, relevance:product.name:0.5:no.
+
+### Match Transducer
+- match.strategy. This can be any of: schema, instances, schema+instances. When the match strategy is set to schema, only attribute names will be compared for similarity. When set to instances, only attribute instances will be compared for similarity. When set to schema+instances, both attribute names and instances will be compared for similarity.
+> The default value is schema.
+
+- match.threshold. Matches scoring below this similarity threshold will be discarded. 
+> The default value is 0.4.
+
+- match.instances.max. The number of instances to compare when evaluating similarity between two attributes. 
```

Comment:
Make wrangle property bold as otherwise the document is hard to read and find what you are looking for.

Reply:
Done! But I'm not sure how to put this as tables because its too much content for a cell